contrib/build-dokku.Dockerfile

```diff
@@ -24,6 +24,7 @@ COPY . ${WORKDIR}
 
 ENV GOPATH=/go
 
+
```

Comment:
Mind removing all these extra newlines you added?

Reply:
Sure can - I added them to make the file a little bit more readable.